### PR TITLE
Add additional permitted characters to Clojure

### PR DESCRIPTION
**Pull request by dadlerj**
Allowed characters (now outdated): https://stackoverflow.com/a/3961674

Arrows: https://stackoverflow.com/questions/21189499/arrow-in-function-name-in-clojure

### CODE REVIEW

File: languages.ts [hunk 1]
```diff
@@ -423,7 +423,7 @@ export const languageSpecs: LanguageSpec[] = [
             languageID: 'clojure',
             fileExts: ['clj', 'cljs', 'cljx'],
             commentStyle: lispStyle,
-            identCharPattern: /[A-Za-z0-9_\-!?]/,
+            identCharPattern: /[A-Za-z0-9_\-!?+*>]/,
```
Comment (chrismwendt) [inline languages.ts]:
```suggestion
            identCharPattern: /[A-Za-z0-9_\-!?+*<>]/,
```

Comment (dadlerj) [reply languages.ts]: Ah, looking at the official list now, I see they've expanded the permitted options (should have checked the source, not just the StackOverflow...). I'll update

File: languages.ts [hunk 2]
```diff
@@ -423,7 +423,7 @@ export const languageSpecs: LanguageSpec[] = [
             languageID: 'clojure',
             fileExts: ['clj', 'cljs', 'cljx'],
             commentStyle: lispStyle,
-            identCharPattern: /[A-Za-z0-9_\-!?]/,
+            identCharPattern: /[A-Za-z0-9_\-!?+*<>]/,
```
Comment (dadlerj) [inline languages.ts]:
```suggestion
            identCharPattern: /[A-Za-z0-9_\-!?+*<>'=]/,
```

File: languages.ts [hunk 3]
```diff
@@ -423,7 +423,7 @@ export const languageSpecs: LanguageSpec[] = [
             languageID: 'clojure',
             fileExts: ['clj', 'cljs', 'cljx'],
             commentStyle: lispStyle,
-            identCharPattern: /[A-Za-z0-9_\-!?]/,
+            identCharPattern: /[A-Za-z0-9_\-!?+*<>'=]/,
```
Comment (dadlerj) [inline languages.ts]:
```suggestion
            identCharPattern: /[A-Za-z0-9_\-!?+*<>=]/,
```

**Pull request merged**